templates/addons/placementGroups/linodePlacementGroup.yaml

```
@@ -0,0 +1,7 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: LinodePlacementGroup
+metadata:
+  name: ${CLUSTER_NAME}
+spec:
+  region: ${LINODE_REGION}
```

can we also add credential refs to this similar to VPC/Firewalls
```suggestion
  credentialsRef:
    name: ${CLUSTER_NAME}-credentials
  region: ${LINODE_REGION}
```